### PR TITLE
challenge mostly complete

### PR DESCRIPTION
**Pull request by JasonDormier**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code

### CODE REVIEW

File: challenges/fifoAnimalShelter/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * dequeue() O(n^2)
+ * made sure everything was setup properly by consoling logging
+ * work through each method one at a time.
+
+
+ ## Collaborators 
+ * Carly Dekock
+ * Jason Quaglia
+ * Seid
+ * Nick Macgruder
```
Comment (adrianhuebner) [inline challenges/fifoAnimalShelter/README.md:24]: Would love to see a whiteboard for challenges like these
